src/util/EventLogger.js

```diff
@@ -136,10 +166,32 @@ export function logPurchaseFlowEvent(
         window.fbq('track', eventNameMapping[event], {
           currency,
           value: price,
-          content_ids: items.map(i => i.classId),
+          content_ids: items.map(
+            i => i.productId || i.collectionId || i.classId
+          ),
         });
       }
     }
+    if (vue.$crisp) {
+      vue.$crisp.push([
+        'set',
+        'session:event',
+        [
+          [
+            [
+              event,
+              {
+                price,
+                currency,
+                items: items.map(
+                  i => i.productId || i.collectionId || i.classId
+                ),
+              },
+            ],
+          ],
+        ],
```

Comment:
3 nested arrays..., is it intentional?

Reply:
Yes... https://docs.crisp.chat/guides/chatbox-sdks/web-sdk/dollar-crisp/#push-session-events